addons/frag/functions/fnc_dev_fired.sqf

```diff
@@ -0,0 +1,19 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add fired rounds to dev track.
+ *
+ * Arguments:
+ * None. Parameters inherited from EFUNC(common,firedEH)
```

Comment:
```suggestion
 * Parameters inherited from EFUNC(common,firedEH)
```

Reply:
Yeah that's fair, I wasn't sure whether to call that "None" or the other part.